### PR TITLE
submission of A0 and A1

### PR DESCRIPTION
**Pull request by cchheerilyn**
my solutions for A0 and A1

### CODE REVIEW

**Comment by cchheerilyn** [comment]
@mtw or @s-will, please review

File: A0/cchheerilyn-HelloWorld.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/bash
+
+echo "Hello, world!"
```
Comment (s-will) [inline A0/cchheerilyn-HelloWorld.sh]: Be precise. According to my specification, this should print "Hello World!"

File: A0/cchheerilyn-HelloWorld.sh [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+cat $1
```
Comment (s-will) [inline A0/cchheerilyn-HelloWorld.sh]:
```suggestion
cat "$1"
```

File: A1/cchheerilyn-WordCount.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
+
+print ("A1 - Word Count")
+
+#For programming warm up, here is the "Word Count" challenge:
+#Write a program that reads in a text (file name given on the command line) and counts the total number of words and the number of different words. On request, the program should print a list of the words.
+
+wordcount = 0
+
+file = input("Please enter a filename: ")
```
Comment (s-will) [inline A1/cchheerilyn-WordCount.py]: One must parse the command line arguments to get the filename; have a look into Python's argparse module

File: A1/cchheerilyn-WordCount.py [hunk 4]
```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
+
+print ("A1 - Word Count")
```
Comment (s-will) [inline A1/cchheerilyn-WordCount.py]: This should not be printed according to my specification; we will always specify exactly what the programs should read and write

File: A1/cchheerilyn-WordCount.py [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+for line in f:
+    small = f.lower()
+    for char in '"\',.:;?!%^()[]{}|-_+=$<>\n':
+        text = small.replace (char, ' ')
+
+#counts and prints total word count
+for line in f:#.lower():
+    text = line.split()
+    wordcount += len(text)
+print("total word count: \t", wordcount)
```
Comment (s-will) [inline A1/cchheerilyn-WordCount.py]: again, check the specification in which format this should be printed

File: A1/cchheerilyn-WordCount.py [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+#counts and prints unique word count
+def uniquewordcount (input):
+    uniques = {}
+    for word in input:
+        if word in uniques:
+            uniques [word] += 1
+        else:
+            uniques [word] = 1
+    return uniques
+print ("unique words: ", uniquewordcount (text))
```
Comment (s-will) [inline A1/cchheerilyn-WordCount.py]: same here

**Comment by cchheerilyn** [comment]
Dear s-will,

Im sorry I wasnt able to attend todays lecture due to a home emergency. I dont know what the consequences will be but either way I think i‘m a little overwhelmed by the workload/requirements in programming skills as u might have seen in my solution for A1. So I guess it‘s better for me to quit your course.

Kind regards, 
Cherilyn

**Pull request closed without merging**